omdb/models.py

```diff
@@ -80,6 +80,7 @@ class Item(Storage):
         'Runtime',
         'Season',
         'seriesID',
+        'totalSeasons',
```

Comment:
It is two different think. 
The `totalResults`  is the number of the results in `search` and `pages` option. 
The `totalSeasons` is how many seasons for the request series.